security/backends/elasticsearch/writer.py

```diff
@@ -66,11 +77,14 @@ def get_queryset_dict_data_by_batch(qs, batch_size):
             yield obj.to_dict()
 
 
-def lazy_serialize_iterable(iterable):
+def get_lazy_serialize_iterable(iterable, stdout, batch):
 
     class StreamList(list):
+
         def __iter__(self):
-            for obj in iterable:
+            for i, obj in enumerate(iterable, 1):
+                if i % batch == 0:
+                    stdout.write('.', ending='')
```

Comment:
write backup file is in batch so "." is progress bar

![image](https://user-images.githubusercontent.com/424409/182929376-6c0e2abb-76e2-476d-95c4-ec198fd6756f.png)